src/tests/ftest/rebuild/read_array.yaml

```diff
@@ -1,63 +1,30 @@
 hosts:
-  test_machines:
-    test_servers:
-      - boro-A
-      - boro-B
-      - boro-C
-      - boro-D
-      - boro-E
-      - boro-F
-    test_clients:
-      - boro-H
-  test_server_count: !mux
-    three_servers:
-      server_count: 3
-    six_servers:
-      server_count: 6
-  test_client_count:
-    one_client:
-      client_count: 1
+  test_servers:
+    - boro-A
+    - boro-B
+    - boro-C
+    - boro-D
+    - boro-E
+  test_clients:
+    - boro-F
+  server_count: 5
+  client_count: 1
 timeout: 240
 server_config:
   name: daos_server
+  server:
```

Comment:
Shouldn't this be changed to "servers" as per PR #1147

Reply:
The problem is #1147 hasn't merged yet, so the yaml key cannot be changed without the accompanying change to `create_server_yaml`.